_includes/masthead

```diff
@@ -3,7 +3,7 @@
 <div id="masthead-no-image-header">
 	<div class="row">
 		<div class="small-12 columns">
-			<a id="logo" href="{{ site.url }}" title="{{ site.title }} – {{ site.slogan }}">
+			<a id="logo" href="{{ site.url }}/" title="{{ site.title }} – {{ site.slogan }}">
```

Comment:
Looks OK, though it would be more correct simply to use href="/" - that way we don't run into issues running on different sites.